src/modules/oauth/services/AuthService.js

```diff
@@ -0,0 +1,59 @@
+import { connect } from 'react-redux';
+import axios from 'axios';
+import queryString from 'query-string'
+import Service from '../../../abc/Service';
+
+
+
+export default class AuthService extends Service {
```

Comment:
There is already `class AuthService` defined ... this is rather `OAuthService`?

Reply:
Maybe?
My idea was, that rest of the application would be apologetic on type of authorization. 
Any form of authorization module would expose AuthService and you could basicly plug and play any type of authorization module in to your app without changing a thing.

Thats why I am using generic name AuthorizationModule as well.

Is that a wrong way to think about it?

Reply:
Name has to be unique ... you are mixing type/class and name.
Simply said, there should be only one class of a given name.

Reply:
Ok, this seems to be a last bit prior merge.